contrib/docker/Dockerfile

```diff
@@ -1,4 +1,5 @@
 FROM ubuntu:14.04
```

Comment:
Just tried the Debian image: the difference is not that big - the Ubuntu one has 180 Mb, the Debian one 130Mb. But the Debian one has a lot of stuff missing, including sudo :) and the proper apt sources files. So getting it to work is a bit of a PITA, while on Ubuntu it 'just worked'. So I would say we leave it on Ubuntu for now, as the time wasted is not worth the 50Mbs. And you know what they say, premature optimisation is the root of all evil ;)